### ISSUE TITLE
flowable 6.4.0 Oracle ExecutionEntityImpl.deleteExecution-Inline ORA-02292

### ISSUE THREAD

**Issue by daitianci** [flowable/flowable-engine#1962]
### Cause: java.sql.SQLIntegrityConstraintViolationException: ORA-02292: integrity constraint (CICRO.FK8A82499F4C67C41) violated - child record found (XXX.SYS_C0047638)

**Comment by daitianci** [comment]
SYS_C0047638 is table ACT_RU_VARIABLE keys ,type Foreign,colums EXECUTION_ID_

**Comment by tijsrademakers** [comment]
Did you reproduce this with Flowable 6.4.2 as well? If so, can you explain in which context you are receiving this exception? Are you able to reproduce it in a unit test?

**Comment by daitianci** [comment]
> Did you reproduce this with Flowable 6.4.2 as well? If so, can you explain in which context you are receiving this exception? Are you able to reproduce it in a unit test?

thanks you for your help.
we use loadrunner to simulate varying concurrent user load.this situation can not be reproduce in a unit test.It will be reproduce by loadrunner

**Comment by tijsrademakers** [comment]
Yes I understand, but please describe what you are testing in Loadrunner exactly, and at which point this exception is happening.
